Library/Homebrew/test/utils/git_spec.rb

```diff
@@ -0,0 +1,209 @@
+require "utils/git"
+
+describe Git do
+  before(:all) do
+    git = HOMEBREW_SHIMS_PATH/"scm/git"
+    file = "lib/blah.rb"
+    repo = Pathname.new("repo")
+
+    (repo/"lib").mkpath
+    system "#{git} init"
+    FileUtils.touch("repo/#{file}")
+
+    File.open(repo/file, "w") { |f| f.write("blah") }
```

Comment:
`(repo/file).write("blah")`. That doesn't require an explicit close because it's handled automatically by `Pathname#write`.